docs/components/DocSearch.js

```diff
@@ -42,6 +40,9 @@ function handleKeyDown(event: KeyboardEvent) {
 
 export default function DocSearch(): Node {
   useEffect(() => {
+    if (typeof window === 'undefined' || !window.docsearch) {
```

Comment:
curious about removing this. why?

Reply:
@AlbertCarreras yes, just being extra safe here to now throw any exceptions.